### DISCUSSION TITLE
Complete the lyrics 🎤

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#533] [Galaxy Trivia] [answered]
```
Is this the real life?
Is this just fantasy?
Caught in a landside,
No escape from ________
Open your eyes,
Look up to the skies and see,
I'm just a poor boy, I need ________,
Because I'm easy come, easy go,
Little high, little low,
Any way the wind blows ________,

Mamaaa,
Just killed a man,
Put a gun against his head, 
Pulled my trigger, now ________
Mamaaa, life had just begun,
But now I've gone and ________
Mama, oooh,
Didn't mean to ________,
If I'm not back again this time tomorrow,
________ as if nothing really matters

Too late, my time has come,
Sends shivers down my spine, body's ________
Goodbye, everybody, I've got to go,
Gotta leave you all behind and ________
Mama, oooh
I don't want to die,
I sometimes wish I'd never ________.
```

**Comment by Veercodeprog** [comment]
Is this the real life?
Is this just fantasy?
Caught in a landside,
No escape from reality
Open your eyes,
Look up to the skies and see,
I'm just a poor boy, I need no sympathy,
Because I'm easy come, easy go,
Little high, little low,
Any way the wind blows doesn't really matter to
Me, to me
Mamaaa,
Just killed a man,
Put a gun against his head, pulled my trigger,
Now he's dead
Mamaaa, life had just begun,
But now I've gone and thrown it all away
Mama, oooh,
Didn't mean to make you cry,
If I'm not back again this time tomorrow,
Carry on, carry on as if nothing really matters
Too late, my time has come,
Sends shivers down my spine, body's aching all
The time
Goodbye, everybody, I've got to go,
Gotta leave you all behind and face the truth
Mama, oooh
I don't want to die,
I sometimes wish I'd never been born at all.

**Comment by AlienegraGeek** [comment, marked as answer]
Bohemian Rhapsody - Queen

Is this the real life?
Is this just fantasy?
Caught in a landslide,
No escape from reality.
Open your eyes,
Look up to the skies and see,
I'm just a poor boy, I need no sympathy,
Because I'm easy come, easy go,
Little high, little low,
Any way the wind blows, doesn't really matter to me, to me.

Mama, just killed a man,
Put a gun against his head,
Pulled my trigger, now he's dead.
Mama, life had just begun,
But now I've gone and thrown it all away.
Mama, ooh,
Didn't mean to make you cry,
If I'm not back again this time tomorrow,
Carry on, carry on, as if nothing really matters.

Too late, my time has come,
Sent shivers down my spine,
Body's aching all the time.
Goodbye, everybody, I've got to go,
Gotta leave you all behind and face the truth.
Mama, ooh
I don't wanna die,
I sometimes wish I'd never been born at all.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/AlienegraGeek?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
